### ISSUE TITLE
🏴Format output code with black

### ISSUE THREAD

**Issue by jfthuong** [unpackAI/unpackai#11]
To have a nice looking code, format the output of `nbdev` with `black` (https://github.com/psf/black).

It should be automatic (Github action ... or pre-commit action ... or a build procedure that runs nbdev and black after that)

**Comment by raynardj** [comment]
## A separate requirements.txt
Shall we put a dev directory? with a README.md inside as guidance, with a requirements.txt as installation instruction, to install things only for dev environment?

As I can imagine for now the requierments will contain:
* nbdev
* black

**Comment by jfthuong** [comment]
yes, we could have a `requirements-dev.txt` .... on the other hand, `nbdev` is supporting requirements for dev in the ini file. To be explored
